docs/source/services/timelock_service/reverse-migration.rst

```diff
@@ -44,10 +44,10 @@ of timestamps across TimeLock and the key-value service backed services is monot
 This can be obtained using ``curl`` as follows. Note that if you obtain an HTTP 503 error, this is because you curled a
 node that was not the leader; please try the other nodes.
 
-.. code-block:: none
+   .. code:: bash
 
-   $ curl -XPOST https://<timelock-host>:8421/timelock/api/client/timestamp/fresh-timestamp
-   123456789
+      curl -XPOST <protocol>://<host>:<port>/timelock/api/tl/ts/<namespace> \
+        --data '{"numTimestamps": 1}' -H "Authorization: Bearer q" -H "Content-Type: application/json"
```

Comment:
possibly also a reminder about the leader here

Reply:
I think Grgur means below, but yep